static/src/js/util/request/exportGranuleSearchRequest.js

```diff
@@ -0,0 +1,17 @@
+import GranuleRequest from './granuleRequest'
+
+export default class ExportGranuleSearchRequest extends GranuleRequest {
+  constructor(authToken, earthdataEnvironment) {
+    super(authToken, earthdataEnvironment)
+    if (authToken && authToken !== '') {
+      this.searchPath = 'granules/export'
+    } else {
+      this.searchPath = 'search/granules.stac'
+    }
+  }
+
+  transformResponse(data) {
```

Comment:
This is redundant, this is just calling the parent which is what would happen if this method did not exist.

Reply:
This is overwriting the transformations that happen in GranuleRequest.transformResponse. We do a lot of tweaks to that response that we don't want to happen here.